packages/mui-material-nextjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/material-nextjs",
-  "version": "6.0.0-alpha.1",
+  "version": "6.0.0-alpha.2",
```

Comment:
Shouldn't this be `6.0.0-alpha.11`?

Reply:
The versions have been incremented from the last published version.

Reply:
Yes but: https://github.com/mui/material-ui/blob/HEAD/scripts/README.md#release:~:text=Keep%20the%20package%20versions%20of%20stable%20public%20packages%20the%20same%20as%20the%20root%20package.json%20version.

Reply:
It should be `6.0.0-alpha.11`.